src/layouts/MainLayout.js

```diff
@@ -110,11 +110,17 @@ const MainLayout = ({ user, children, loginCallback }) => {
                           Your Cubes
                         </DropdownToggle>
                         <DropdownMenu right>
-                          {user.cubes.map((item) => (
-                            <DropdownItem key={`dropdown_cube_${item.name}`} href={`/cube/overview/${item._id}`}>
-                              {item.name}
-                            </DropdownItem>
-                          ))}
+                          <>
```

Comment:
Is there a reason this fragment needs to be here?

Reply:
It's need, a root element can only contain one jsx tag

Reply:
NVM: was looking at the wrong one, this can be removed